Add support for a custom block name and migrations to `createRichTextBlock` (#1544)
vivid-planet/comet: .changeset/hungry-wombats-behave.md

---
"@comet/blocks-api": minor
---

> Add property `migrationOptions` to `createRichTextBlock()` factory to enable adding migrations to the RichTextBlock

Reviewer: ```suggestion
Add support for a custom block name and migrations to `createRichTextBlock`
```

> Add property migrationOptions to createRichTextBlock factory in order to be able to add migrations to the RichTextBlock

Reviewer 2: ```suggestion
Add property `migrationOptions` to `createRichTextBlock()` factory to enable adding migrations to the RichTextBlock
```
